### DISCUSSION TITLE
toy data

### DISCUSSION THREAD

**Discussion by Kurtanjek** [jakobrunge/tigramite#397] [General]
Can you advise how to resolve:
T = 100

data, true_parents_neighbors =toys.structural_causal_process(links_coeffs, T=T)
Traceback (most recent call last):

  Cell In[20], line 1
    data, true_parents_neighbors =toys.structural_causal_process(links_coeffs, T=T)

  File ~\anaconda3my\lib\site-packages\tigramite\toymodels\structural_causal_processes.py:717 in structural_causal_process
    func = link_props[2]

IndexError: tuple index out of range

**Comment by jakobrunge** [comment]
That was wrong in the medium tutorial, now fixed to: 

def lin_f(x): return x # A linear dependency function
links_coeffs = {0: [((0, -1), 0.7, lin_f), ((1, -1), -0.8, lin_f)],
                1: [((1, -1), 0.8, lin_f), ((3, -1), 0.8, lin_f)],
                2: [((2, -1), 0.5, lin_f), ((1, -2), 0.5, lin_f), ((3, -3), 0.6, lin_f)],
                3: [((3, -1), 0.4, 
                } #stores the functional dependencies of the SCM
